docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.1'
+services:
+  postgres:
+    environment:
+      - POSTGRES_DB=osso
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=password
```

Comment:
not sure really what the ideal solution here is.

is the idea that the password isnt really secret because only services in the container would have access to it? just feels bit circular to stick these things in a .env file. and then how would a user specify those when pulling / deploying an image?